### ISSUE TITLE
For Civiform admin, allow ability to toggle on and off autopopulate feature

### ISSUE THREAD

**Issue by lianne-oy** [seattle-uat/civiform#1113]
**Is your feature request related to a problem? Please describe.**
When asking for "signature" or related approval fields we should not autopopulate forms for applicants, even if previously submitted for other programs.  

**Describe the solution you'd like**
Ability for CF admin to remove auto populate feature.

**Comment by neel-a-mehta** [comment]
Good point -- so the CiviForm admin, when building a question, can check a certain checkbox that bans auto-filling of that question.  So even if we have the applicant's data for that program, we shouldn't auto-fill it.

**Comment by elisekalstad** [comment]
Work-around: create a new question that is unique to the program. Could pose issues with the reapply process though.

**Comment by msprenke** [comment]
This is necessary for any kind of recertification purpose and may be needed for UAPA integration and would be extremely useful for people who apply on behalf of family members. This is not a TI. It is not uncommon for people to apply for someone else due to language, age, or disability. 

If we ask a question at the beginning, are you applying on behalf of someone else then it stops the auto populate

**Comment by msprenke** [comment]
required for DEEL onboarding
